recipes/bitsery/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from conans import ConanFile, tools
+
+
+class BitseryConan(ConanFile):
+    name = "bitsery"
+    description = "Header only C++ binary serialization library. It is designed around the networking requirements for real-time data delivery, especially for games."
+    topics = "serialization", "binary", "header-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/fraillt/bitsery"
+    license = "MIT"
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        tools.check_min_cppstd(self, "11")
```

Comment:
```suggestion
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, "11")
```

Reply:
Why is for C++11 no compiler check required?